packages/cli/src/commands/prisma.js

```diff
@@ -0,0 +1,73 @@
+import fs from 'fs'
+
+import execa from 'execa'
+import terminalLink from 'terminal-link'
+
+import { getPaths } from '@redwoodjs/internal'
+
+import c from 'src/lib/colors'
+
+export const command = 'prisma [...commands]'
+export const description = 'Run Prisma CLI with built-in helpers'
+
+export const builder = (yargs) => {
+  const argv = process.argv.slice(3)
+  const paths = getPaths()
+
+  // descriptions below generate list when `rw prisma --help` is run
+  yargs
+    .example("'yarn rw prisma migrate dev'", 'No additional options required')
+    .positional('db push', {
+      alias: 'push',
+      description:
+        'Push the state from your Prisma schema to your database without creating a migration',
+    })
+    .positional('db seed', {
+      description: "Seed your database. Requires a 'seed.js' file",
+    })
+    .positional('migrate dev', {
+      description:
+        'Create a migration from schema.prisma; apply it to the dev database',
```

Comment:
If the user adds `--create-only` as a flag, then it does not apply it.

Has a `migrate dev --create-only` alias been considered?

I like to review the SQL before I apply -- to double-check for mistakes and that I like the SQL it generated. There may be times I would tweak it I imagine before I apply?

Reply:
The `rw prisma ...` command is just a wrapper that does a touch of special handling but mostly passes through the flags to run the Prisma CLI. So anything you can do with Prisma, you can do here.

<img width="325" alt="Screen Shot 2021-02-05 at 8 28 16 AM" src="https://user-images.githubusercontent.com/2951/107060663-19bf9380-678c-11eb-9315-fedfc68d9cf8.png">
<img width="409" alt="Screen Shot 2021-02-05 at 8 28 39 AM" src="https://user-images.githubusercontent.com/2951/107060713-26dc8280-678c-11eb-813d-0138a27afe9c.png">